morning_session.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'session'
+
+class MorningSession < Session
```

Comment:
this inheritance is not required as the same can be acheived via an enum type in session.